experiments/example/example_with_circuit_breaker.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
-$LOAD_PATH.unshift(File.expand_path("../lib", __dir__))
+$LOAD_PATH.unshift(File.expand_path("../../lib", __dir__))
```

Comment:
let's just delete the example. It's not up-to-date with how we're doing most things now, and we do have a ton of real tests instead of the example